const handleLogin = async () => {
  try {
    console.log('Logging in...');
    await app.auth().signInWithEmailAndPassword(email, password);
    console.log('User successfully logged in');
  } catch (error) {
    console.error('Error logging in:', error);
  }
};

const handleRegister = async () => {
  try {
    console.log('Registering...');
    await app.auth().createUserWithEmailAndPassword(email, password);
    console.log('User successfully registered');
  } catch (error) {
    console.error('Error registering:', error);
  }
};
